array-manipulation-3/src/equal.js

```diff
@@ -0,0 +1,14 @@
+/* exported equal */
+function equal(first, second) {
+  for (var i = 0; i < first.length; i++) {
+    if (first[i] !== second[i]) {
+      return false;
+    }
+  }
+  for (var j = 0; j < second.length; j++) {
+    if (second[i] !== first[i]) {
+      return false;
+    }
+  }
```

Comment:
This second loop is checking the same thing as the first loop so isn't necessary